base/applications/taskmgr/graphctl.c

```diff
@@ -407,6 +419,52 @@ void GraphCtrl_Paint(TGraphCtrl* this, HWND hWnd, HDC dc)
     DeleteDC(memDC);
 }
 
+void GraphCtrl_ShiftGrid(TGraphCtrl* this)
+{
+    RECT rectCleanUp;
+    HPEN oldPen;
+    HPEN solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
```

Comment:
Suggestions:
Move `CreatePen()`inside `if (this->m_dcGrid != NULL)`.
Check its result?

Reply:
Task manager doesn't check CreatePen result anywhere else... So it'd be another story for this? Will have to add a whole bunch of checks.

Reply:
Agreed then: skip adding that check.

Reply:
Most of the code were just copy-pasted inside task manager so I don't really know what could I do about these existing code 😂